### DISCUSSION TITLE
HELP!!! Kadjar car radio unit change

### DISCUSSION THREAD

**Discussion by dinukro** [cedricp/ddt4all#949] [Q&A]
Hi everyone, I tried to update the software version of my car radio, but unfortunately the car radio is blocked (black screen).
I bought a second hand car radio but unfortunately it has anti-theft protection.
Please help me fix this problem.

**Comment by matzel82** [comment]
Black display have to need reset and autoconfig with Renault original clip....

**Reply by dinukro** [reply]
Hi thank you for quick response, I went to Renault but they told me that it is most likely broken and it costs 500 euros to change it, for this reason I bought a used one. There is any other solution ?
Thank you

**Reply by matzel82** [reply]
That's possible... "I" know this problem after update. Sometimes I have to rescued the RLink

**Reply by dinukro** [reply]
Sorry I'm not understand, you can help me to save the problem?

**Comment by AndyIreland** [comment]
> Hi everyone, I tried to update the software version of my car radio, but unfortunately the car radio is blocked (black screen). I bought a second hand car radio but unfortunately it has anti-theft protection. Please help me fix this problem.

Have a read through this and see if it helps:
https://www.drive2.ru/b/643019689764615344/
Restoring R-Link1 and R-Link2 after resetting the configuration

and you could also try ask your question on here too:
https://www.facebook.com/groups/764193797465935
